src/mame/drivers/firebeat.cpp

```diff
@@ -2355,10 +2355,10 @@ ROM_START( kbh )
 	ROM_LOAD("gu974", 0x00, 0xc8, CRC(748b8476) SHA1(5d507fd46235c4315ad32599ce87aa4e06642eb5))
 
 	DISK_REGION( "ata:0:cdrom" ) // program CD-ROM
-	DISK_IMAGE_READONLY( "gu974-ka a01", 0, SHA1(af4e8182f6a984895d9a9a00bbfb6c65fb7b4738) )
+	DISK_IMAGE_READONLY( "gu974-ka a01", 0, SHA1(07d3d6abcb13b2c2a556f2eed7e89e3d11febf1b) )
 
 	DISK_REGION( "ata:1:cdrom" ) // audio CD-ROM
-	DISK_IMAGE_READONLY( "gu974-ka a02", 1, SHA1(e66930f965b1aa1a681ab696302a04958dc8a334) ) // identical to jaa02 image
```

Comment:
The .img is identical but the subchannel data isn't.
```sh
> sha1sum -b GQ974-JAA02/GQ974-JA\ A02.img GQ974-JAA02/GQ974-JA\ A02.sub GU974-KAA02/GU974-KA\ A02.img GU974-KAA02/GU974-KA\ A02.sub 
cc7d90d902c0e5553988149bc57a05c3cac472b5 *GQ974-JAA02/GQ974-JA A02.img
7f3a2f6a19bdc01bee2e090e696b3c3c6b6b2056 *GQ974-JAA02/GQ974-JA A02.sub
cc7d90d902c0e5553988149bc57a05c3cac472b5 *GU974-KAA02/GU974-KA A02.img
ec9c52ebcd1a769748628a4ea6d334e93cbed9ae *GU974-KAA02/GU974-KA A02.sub
```

Reply:
subchannel data tends to be less repeatable on multiple reads, so it's possible the discs are still identical, just you'd get slightly different subchannel data each time, it's one of the issues with CD dumping (and why in some cases, it isn't included unless it's needed for protections etc.)

Reply:
That’s why we have things like cdparanoia for dealing with it.  What tools were actually used to produce the dumps in the first place?

Reply:
afaik for subchannel data even tools like cdparanoia don't really help, there are long and heated debated on the subject to the point people have literally just given up, because it became a blocking issue.